src/QsCompiler/LlvmBindings/LLVMSharpExtensions/LLVMMetadataRefExtensions.cs

```diff
@@ -0,0 +1,160 @@
+﻿// <copyright file="LLVMMemoryBufferRefExtensions.cs" company="Ubiquity.NET Contributors">
+// Copyright (c) Ubiquity.NET Contributors. All rights reserved.
+// Portions Copyright (c) Microsoft Corporation
+// </copyright>
+
+using System;
+
+namespace LLVMSharp.Interop
+{
+    /// <summary>Extensions for <see cref="LLVMMetadataRef"/>.</summary>
+    public static unsafe class LLVMMetadataRefExtensions
+    {
+        /// <summary>Convenience wrapper for <see cref="LLVM.GetMetadataKind"/>.</summary>
+        public static LLVMMetadataKind GetMetadataKind(this LLVMMetadataRef self)
+        {
+            return (LLVMMetadataKind)LLVM.GetMetadataKind(self);
+        }
+
+	/// <summary>Convenience wrapper for <see cref="LLVM.MetadataReplaceAllUsesWith"/>.</summary>
```

Comment:
Looks like there are a few places where tabs snuck in instead of spaces... hopefully the autoformat will take care of that, but if not please switch to spaces.